static/about.html

```diff
@@ -0,0 +1,42 @@
+<html>
+<head>
+</head>
+
+<body>
+
+<header>
+  <h1> Wang </h1>
+
+<nav>
```

Comment:
Good use of nav! It gives the links some semantic relevance and gives you a hook for styling later.